raiden-dapp/src/store/index.ts

```diff
@@ -131,7 +141,21 @@ const store: StoreOptions<RootState> = {
         value => value.state === ChannelState.open
       );
       return orderBy(openChannels, ['capacity'], ['desc'])[0];
-    }
+    },
+    pendingTransfers: ({ transfers }: RootState) =>
+      Object.keys(transfers)
+        .filter(secretHash => {
+          const { status } = transfers[secretHash];
+          const isTransferPending =
+            status !== RaidenSentTransferStatus.unlocked && // success completed case
```

Comment:
Actually, pending transfers are `!transfer.completed`

Reply:
From #585 :

> Success case: PENDING -> RECEIVED -> REVEALED -> UNLOCKING -> UNLOCKED (completed, succeeded)
> Example of error case (e.g. mediators take more fees than they can): PENDING -> RECEIVED (2*reveal blocks later) EXPIRING -> EXPIRED (completed, failed)

Reply:
Yes, these are information for the stepper (when/if it's implemented) of the transfer workflow. The `completed` and `success` properties of this type were created specifically for this filtering/showing, and consider some other conditions as well (e.g. transfer refunded, channel closed, etc).

Reply:
Then they should be mentioned in the ticket, not information for steppers that are not in scope at all